ext/reflection/php_reflection.stub.php

```diff
@@ -476,6 +476,10 @@ public function getDeclaringClass() {}
     /** @return string|false */
     public function getDocComment() {}
 
+    public function hasType(): bool {}
+
+    public function getType(): ?ReflectionType {}
```

Comment:
Should we add tests for these new methods?

Reply:
Good catch! Will add those when I have some more time.